### ISSUE TITLE
[Bug Report] crash after type command 'exit'

### ISSUE THREAD

**Issue by zombie110year** [microsoft/terminal#1275]
# Environment

```none
Windows build number:  10.0.18917.1000
Windows Terminal version: commit 315abf6fa6e096f0ad8ba7db68fa8a3fd9a591b1
```

# Steps to reproduce

1. start Terminal application
2. run some commands in the prompt
3. run command `exit`

# Expected behavior

Simply exit shell.

# Actual behavior

![image](https://user-images.githubusercontent.com/36528430/59549389-5fb05e00-8f8f-11e9-8fcc-2018ce0b5119.png)

Run `exit` command straightly didn't cause a crash, but it will happen after run any other command. It happens in all the default `PowerShell`, `cmd`, `WSL` profiles.

the profiles JSON file's modifications are:

```diff
--- profiles.old.json	2019-06-15 17:07:10.565458400 +0800
+++ profiles.json	2019-06-15 17:07:06.089459800 +0800
@@ -1,10 +1,10 @@
 {
     "globals" :
     {
-        "alwaysShowTabs" : true,
+        "alwaysShowTabs" : false,
         "defaultProfile" : "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
-        "initialCols" : 120,
-        "initialRows" : 30,
+        "initialCols" : 80,
+        "initialRows" : 24,
         "keybindings" :
         [
             {
@@ -199,28 +199,28 @@
         ],
         "requestedTheme" : "system",
         "showTabsInTitlebar" : true,
-        "showTerminalTitleInTitlebar" : true
+        "showTerminalTitleInTitlebar" : false
     },
     "profiles" :
     [
         {
-            "acrylicOpacity" : 0.5,
-            "background" : "#012456",
+            "acrylicOpacity" : 0.30000001192092896,
+            "background" : "#282C34",
             "closeOnExit" : true,
-            "colorScheme" : "Campbell",
+            "colorScheme" : "One Half Dark",
             "commandline" : "powershell.exe",
             "cursorColor" : "#FFFFFF",
             "cursorShape" : "bar",
-            "fontFace" : "Consolas",
+            "fontFace" : "Sarasa Term SC",
             "fontSize" : 10,
             "guid" : "{61c54bbd-c2c6-5271-96e7-009a87ff44bf}",
             "historySize" : 9001,
             "icon" : "ms-appx:///ProfileIcons/{61c54bbd-c2c6-5271-96e7-009a87ff44bf}.png",
             "name" : "Windows PowerShell",
-            "padding" : "0, 0, 0, 0",
+            "padding" : "4, 4, 4, 4",
             "snapOnInput" : true,
             "startingDirectory" : "%USERPROFILE%",
-            "useAcrylic" : false
+            "useAcrylic" : true
         },
         {
             "acrylicOpacity" : 0.75,
```

**Comment by nntantidakis** [comment]
I can confirm this problem as well.. Whenever the shell exits for some reason, the app becomes totally unresponsive without any modification to the file profiles.json

**Comment by DHowett-MSFT** [comment]
I think this is the same as #1250.

**Comment by DHowett-MSFT** [comment]
Yeah, it looks like a dupe.

**Comment by zombie110year** [comment]
Confirmed resolved, lastest commit 440bee0e4abd0a778cbdff5b86f73fb34a92f3ed.
